docs/BUILD

```diff
@@ -9,20 +9,24 @@ exports_files(["protodoc_manifest.yaml"])
 
 envoy_package()
 
-# TODO(phlax): fix failing/excluded configs
-# the following config only fails on windows:
-#       dns-cache-circuit-breaker: "Error: unable to read file: /etc/ssl/certs/ca-certificates.crt"
-
 filegroup(
     name = "configs",
     srcs = glob(
         ["root/**/*.yaml"],
         exclude = [
+            # `some.customer.filter`
             "root/intro/_include/life-of-a-request.yaml",
+            # "Error: No TLS certificates found for server context"
             "root/intro/arch_overview/security/_include/ssl.yaml",
-            "root/configuration/http/http_filters/_include/dns-cache-circuit-breaker.yaml",
-            "root/configuration/http/http_filters/_include/grpc-reverse-bridge-filter.yaml",
+            # missing proto.pb
```

Comment:
im wondering if/how we can include this `proto.pb` file into the test bundle

Reply:
I think this can be a bazel file dep? cc @htuch @lizan

Reply:
Yep I guess this need to be part of artifact, or bazelify whole of example test (which is out of scope of this PR)

Reply:
this should be fixed. i generated a `.pb` file and added to bazel build etc

i have also updated the docs, as the required steps were not very clear, and the examples were a bit inconsistent